components/protobuf/protobuf.p5m

```diff
@@ -0,0 +1,108 @@
+file path=usr/include/google/protobuf/message.h
+file path=usr/include/google/protobuf/message_lite.h
+file path=usr/include/google/protobuf/reflection_ops.h
+file path=usr/include/google/protobuf/repeated_field.h
+file path=usr/include/google/protobuf/service.h
+file path=usr/include/google/protobuf/stubs/atomicops.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_arm64_gcc.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_arm_gcc.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_arm_qnx.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_atomicword_compat.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_generic_gcc.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_macosx.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_mips_gcc.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_pnacl.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_solaris.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_tsan.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_x86_gcc.h
+file path=usr/include/google/protobuf/stubs/atomicops_internals_x86_msvc.h
+file path=usr/include/google/protobuf/stubs/common.h
+file path=usr/include/google/protobuf/stubs/once.h
+file path=usr/include/google/protobuf/stubs/platform_macros.h
+file path=usr/include/google/protobuf/stubs/stl_util.h
+file path=usr/include/google/protobuf/stubs/template_util.h
+file path=usr/include/google/protobuf/stubs/type_traits.h
+file path=usr/include/google/protobuf/text_format.h
+file path=usr/include/google/protobuf/unknown_field_set.h
+file path=usr/include/google/protobuf/wire_format.h
+file path=usr/include/google/protobuf/wire_format_lite.h
+file path=usr/include/google/protobuf/wire_format_lite_inl.h
+link path=usr/lib/$(MACH64)/libprotobuf-lite.so target=libprotobuf-lite.so.9.0.1
+link path=usr/lib/$(MACH64)/libprotobuf-lite.so.9 \
+    target=libprotobuf-lite.so.9.0.1
+file path=usr/lib/$(MACH64)/libprotobuf-lite.so.9.0.1
+link path=usr/lib/$(MACH64)/libprotobuf.so target=libprotobuf.so.9.0.1
+link path=usr/lib/$(MACH64)/libprotobuf.so.9 target=libprotobuf.so.9.0.1
+file path=usr/lib/$(MACH64)/libprotobuf.so.9.0.1
+link path=usr/lib/$(MACH64)/libprotoc.so target=libprotoc.so.9.0.1
+link path=usr/lib/$(MACH64)/libprotoc.so.9 target=libprotoc.so.9.0.1
+file path=usr/lib/$(MACH64)/libprotoc.so.9.0.1
+file path=usr/lib/$(MACH64)/pkgconfig/protobuf-lite.pc
+file path=usr/lib/$(MACH64)/pkgconfig/protobuf.pc
+link path=usr/lib/libprotobuf-lite.so target=libprotobuf-lite.so.9.0.1
+link path=usr/lib/libprotobuf-lite.so.9 target=libprotobuf-lite.so.9.0.1
+file path=usr/lib/libprotobuf-lite.so.9.0.0
+file path=usr/lib/libprotobuf-lite.so.9.0.1
+link path=usr/lib/libprotobuf.so target=libprotobuf.so.9.0.1
+link path=usr/lib/libprotobuf.so.9 target=libprotobuf.so.9.0.1
+file path=usr/lib/libprotobuf.so.9.0.0
```

Comment:
Couldn't find 'usr/lib/libprotoc.so.9.0.0' in any of the specified search directories.... Please, gmake clean and update manifest